packages/terra-slider/tests/jest/Slider.test.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Slider from '../../src/Slider';
+
+describe('Slider', () => {
+  it('should render a default slider', () => {
+    const wrapper = shallow(
+      <Slider minimumValue={0} maximumValue={100} labelName="testLabel" defaultValue={50} />,
+    );
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render a disabled slider', () => {
+    const wrapper = shallow(
+      <Slider isDisabled minimumValue={0} maximumValue={100} labelName="testLabel" defaultValue={50} />,
+    );
+    expect(wrapper).toMatchSnapshot();
```

Comment:
Can we add an assertion to check the value of the `disabled` attribute on the `input` element?